spk/shairport-sync/src/dsm-control.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+# Package
+PACKAGE="shairport-sync"
+DNAME="Shairport Sync"
+
+# Others
+PORT="4242"
+SHAIRPORT="shairport-sync"
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+DAEMON="${INSTALL_DIR}/bin/${SHAIRPORT}"
+VAR_DIR="${INSTALL_DIR}/var"
+PID_FILE="${VAR_DIR}/${PACKAGE}.pid"
```

Comment:
It's minor, but I thought I already mentioned this: Just use ${INSTALL_DIR} here.